src/components/UserActions/UserActivity.ts

```diff
@@ -276,8 +278,16 @@ export class UserActivity extends Component {
         li.classList.add(VIEW_EVENT_CLASS);
 
         const dataElement = document.createElement('div');
+        if (action.raw.content_id_key === CLICKABLE_URI_ID) {
```

Comment:
content id key is not set to permanent id by default iirc 🤔

Reply:
Wouldn't it compare `undefined` to `clickable_uri_id` in the case that it is not present?

Reply:
@jeremierobert-coveo Je pense pas qu'ont veuille changer ca.. Ce serais pour pouvoir utiliser un autre content id key a afficher comme link ?